Updating to reflect 2023 better (#69)
OctopusDeploy/People: Software-Engineering/Levels-Bands-And-Performance-Reviews.md

Summarising, we want to ensure that:
- Our salaries pass the "printer test" (same job, same pay)
- We support both "superstars" seeking a steep growth trajectory and "rockstars" seeking a shallower growth trajectory
- We have roles and levels that set clear expectations between individuals across Octopus

Octopus People (this repository) is a core building block we use to help people navigate their career at Octopus and one area that deserves a bit more explanation is job levels and bands and how we use performance reviews at Octopus.

In short:
- **Level** (i.e. L1-5) is used to set the person's role title (e.g. Junior Software Engineer, Senior Software Engineer), to broadcast expectations and reinforce archetypes, to the person, across Octopus and beyond.
- **Band** (i.e. Maturing/Performing/Exceeding) is used to reflect value contributed within that level and provide a sense of progression within each level and smoother transitions to the next level

> - **Performance Reviews** are the opportunity for you and your manager will reflect on your demonstrated performance against the expectations of that level + band, and perhaps determine you are ready for a promotion to the next level/band.

Reviewer: ```suggestion
- **Performance Reviews** are an opportunity for you and your manager to reflect on your demonstrated performance against the expectations of that level + band, and determine whether you are ready for a promotion to the next level/band.
```

### Understanding bands

> While levels are pretty easy to understand since they are tied directly to a definition in Octopus People, we've seen some people struggle to calibrate on what the bands mean. This section is designed to help everyone calibrate on the meaning of each band, *maturing*, *performing*, and *exceeding*.

Reviewer: > [from PR description] We've since disconnected the overall rating for performance reviews, using a different set of names and values, so that Levels and Bands can be unambiguously their own thing.

It looks like the overall rating for performance reviews currently still unfortunately has one overlapping name, specifically the `Exceeding` overall rating and the `Exceeding` band.

Sidenote: The overall rating for performance reviews also uses the word "maturity" rather than "band". It would be ideal to choose one and stick to it throughout.

Reviewer: ![image](https://github.com/OctopusDeploy/People/assets/1407488/d80501a6-b3a5-4b70-952e-b629f8035d92)
